css/style.css

```diff
@@ -0,0 +1,103 @@
+body {
+	font-family: "Segoe UI", Arial, sans-serif;
```

Comment:
Something wrong with indentation in that file. Please check it

Reply:
Fixed.